cmd/repo.go

```diff
@@ -101,6 +101,27 @@ func findTemplateURL(projectVersion ProjectVersion, templateName string) string
 	return ""
 }
 
+type indexError struct {
+	indexName string
+	theError  error
+}
+
+type indexErrors struct {
+	listOfErrors []indexError
+}
+
+func (e indexError) Error() string {
+	return "- Repository: " + e.indexName + "\n  Reason: " + fmt.Sprintf("%s", e.theError)
+}
+
+func (e indexErrors) Error() string {
+	var myerrors string
```

Comment:
Can you use the built in join method here? Something like:
`return strings.Join(e.listOfErrors, "\n")`

Reply:
I tried this and I cannot use `e.listOfErrors` with the strings join method as it is not of type `[]string`

Reply:
Of course, nevermind :-)